src/components/common-headers/README.md

```diff
@@ -0,0 +1,55 @@
+<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
+
+<a name="readme-top"></a>
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  
+<h3 align="center">GOV UK One Login Common Headers</h3>
+  <p align="center">
+    This package enables GOV UK LOGIN frontend Node.js applications to add one login common headers.
+    <br />
+    <a href=""><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/govuk-one-login/di-fec-ga4-demo">View Demo</a>
+    ·
+    <a href="https://github.com/govuk-one-login/di-fec-ga4-demo/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/govuk-one-login/di-fec-ga4-demo/issues">Request Feature</a>
```

Comment:
Just a suggestion, but maybe we add a link to our collab slack channel incase people want to get in touch?